kuma/javascript/src/bcd-signal.jsx

```diff
@@ -0,0 +1,829 @@
+// @flow
+import { gettext, interpolate } from './l10n.js';
+import { getCookie } from './utils.js';
+
+let bcSignalsInvoked = false;
+
+export function activateBCDSignals(slug: string, locale: string) {
```

Comment:
Would it be possible to just do:
```suggestion
window.activateBCDSignals = (slug: string, locale: string) => {
```
instead?

Reply:
This is doable. I will check if this works well in a bit.